src/UserService/Controllers/CommunicationController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.UserService.Business.Commands.Communication.Interfaces;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User.Communication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Net;
+
+namespace LT.DigitalOffice.UserService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class CommunicationController : ControllerBase
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public CommunicationController(
+            [FromServices] IHttpContextAccessor httpContextAccessor)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        [HttpPost("create")]
+        public OperationResultResponse<Guid> Create(
+            [FromServices] ICreateCommunicationCommand command,
+            [FromBody] CreateCommunicationRequest request)
+        {
+            OperationResultResponse<Guid> result = command.Execute(request);
+
+            if (result.Status != OperationResultStatusType.Failed)
+            {
+                _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
+            }
+
```

Comment:
i think you must add 409 status if communication.value already exists

Reply:
fixed

Reply:
fixed